src/index.html

```diff
@@ -0,0 +1,90 @@
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css" />
+    <!-- Bootstrap CSS -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+
+    <title>Leaderboard</title>
+  </head>
+  <body>
+    <h1 class="text-center m-5">Leaderboard</h1>
+    <main class="row">
+      <div class="col m-5">
+        <div class="d-flex justify-content-between m-3">
+          <h3 class="text-center">Recent scores</h3>
+          <button class="btn btn-outline-secondary text-dark">refresh</button>
+        </div>
+        <div class="mt-2 border border-2 listbox border-dark">
+          <h5 class="m-2">Name:100</h5>
+          <h5 class="m-2">Name:100</h5>
+          <h5 class="m-2">Name:100</h5>
+          <h5 class="m-2">Name:100</h5>
+          <h5 class="m-2">Name:100</h5>
+          <h5 class="m-2">Name:100</h5>
+          <h5 class="m-2">Name:100</h5>
+          <h5 class="m-2">Name:100</h5>
+          <h5 class="m-2">Name:100</h5>
+          <h5 class="m-2">Name:100</h5>
+        </div>
+
+        <div></div>
+      </div>
+      <div class="col m-5">
+        <h2 class="text-center">Add your score</h2>
+        <form
+          class="mt-5 d-flex justify-content-center flex-column gap-3"
+          action="#"
+          id="form"
+        >
+          <input
+            class="p-2"
+            type="text"
+            name="name"
+            id="name"
+            placeholder="Your name"
+            required
+          />
+          <input
+            class="p-2"
+            type="text"
+            name="name"
+            id="score"
+            placeholder="Your score"
+            required
+          />
+          <button
+            class="btn btn-success text-dark w-25"
+            id="button"
+            type="submit"
+          >
+            Save
+          </button>
+        </form>
+      </div>
+    </main>
+
+    <!-- Optional JavaScript; choose one of the two! -->
+
+    <!-- Option 1: Bootstrap Bundle with Popper -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+      crossorigin="anonymous"
+    ></script>
+
+    <!-- Option 2: Separate Popper and Bootstrap JS -->
+    <!--
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" crossorigin="anonymous"></script>
+    -->
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.